src/core/contexts.ts

```diff
@@ -15,22 +15,20 @@ type UpdateAtom = <Value, Update>(
 type CommitCallback = () => void
 
 type StoreForProduction = [
-  stateMutableSource: MutableSource<State>,
-  updateAtom: UpdateAtom,
-  commitCallback: CommitCallback
+  MutableSource<State>, // stateMutableSource
+  UpdateAtom, // updateAtom
+  CommitCallback, // commitCallback
+  (values: Iterable<readonly [Atom<unknown>, unknown]>) => void // restore
```

Comment:
You don't like named tuples? I think it would show `restore` in the editor. The comment doesn't help to tell what this function is at the caller end.

Reply:
I love named tuples, but I can't reuse the types, if it's a named tuple.

Reply:
oh gosh.